website/docs/ref/macro.md

```diff
@@ -139,6 +139,20 @@ i18n._(
 );
 ```
 
+```js
+t('attachment.saved')`Attachment ${name} saved`;
```

Comment:
Does it allow id to be only StringLiteral ot TplLiteral as well?  If Template literal also possible it should be in tests as well.